cypress/cypress/integration/history.spec.ts

```diff
@@ -23,81 +28,81 @@ describe("History", () => {
 
   it("displays questions that have been asked via input", () => {
     visitAsGuestWithDefaultSetup(cy, "/");
-    cy.get("[data-cy=header]").should("have.attr", "data-mentor", "clint");
-    cy.get("[data-cy=input-field]").type("Hello");
-    cy.get("[data-cy=input-send]").trigger("mouseover").click();
-    cy.get("[data-cy=topic-2]").trigger("mouseover").click();
-    cy.get("[data-cy=scrolling-questions-list]").contains("Hello");
+    // cy.get("[data-cy=header]").should("have.attr", "data-mentor", "clint");
```

Comment:
delete this whole `history.spec.ts` file if it's now replaced by `history-chat` or better: rename your `history-chat.spec.ts` to `history.spec.ts` since you're really updating the history behavior

Reply:
OK, the new test file contains all the old tests except for "displays most recent questions at the top and does not read duplicate questions"  according to what we talked about yesterday in the standup. I already rename `history-chat.spec.ts` -> `history.spec.ts`